pkg/clusteragent/admission/mutate/config.go

```diff
@@ -118,6 +142,23 @@ func injectConfig(pod *corev1.Pod, _ string, _ dynamic.Interface) error {
 
 	injectedEntity = injectEnv(pod, ddEntityIDEnvVar)
 
+	// Inject env variables used for Onboarding KPIs propagation
+	if isApmInstrumentationEnabled(pod.Namespace) {
+		// if Single Step Instrumentation is enabled, inject DD_INSTRUMENTATION_INSTALL_TYPE:k8s_single_step
+		_ = injectEnv(pod, singleStepInstrumentationInstallTypeEnvVar)
+	} else {
+		// if local library injection is enabled, inject DD_INSTRUMENTATION_INSTALL_TYPE:k8s_lib_injection
+		_ = injectEnv(pod, localLibraryInstrumentationInstallTypeEnvVar)
+	}
+	// inject DD_INSTRUMENTATION_INSTALL_TIME with current Unix time
+	_ = injectEnv(pod, instrumentationInstallTimeEnvVar)
+	// inject DD_INSTRUMENTATION_INSTALL_ID with UUID created during the Agent install time
+	instrumentationInstallIDEnvVar := corev1.EnvVar{
+		Name:  instrumentationInstallIDEnvVarName,
+		Value: os.Getenv(instrumentationInstallIDEnvVarName),
```

Comment:
Just confirming that this environment variable is going to be set with changes to the Helm chart that will be a part of a different PR?